src/Sfa.Tl.Matching.Web/Views/Opportunity/_placementInformation.cshtml

```diff
@@ -5,15 +5,38 @@
     <tbody class="govuk-table__body">
         <tr class="govuk-table__row">
             <th class="govuk-table__header" scope="row">Type of placement</th>
-            <td class="govuk-table__cell">@Model.RouteName</td>
+            <td class="govuk-table__cell">
+                @Model.RouteName
+            </td>
+            <td class="govuk-table__cell">
+                <a asp-route="ProviderResults_Get" 
+                   asp-route-opportunityId="@Model.OpportunityId" 
+                   asp-route-postcode="@Model.Postcode" 
+                   asp-route-selectedRouteId="@Model.RouteId" 
+                   class="govuk-link tl-float-right">Change<span class="govuk-visually-hidden"> the type of placement</span></a>
```

Comment:
whats the point of having this hidden span?

Reply:
From Ash - "It's for screen readers
So when they're reading the content without visual clues it still makes sense
Otherwise tabbing through it'll just say "change" and not specify what it's doing"